aggregator/aggregator.go

```diff
@@ -159,10 +165,18 @@ func (a *Aggregator) processAndSubmitBatch(txs []core.Tx) {
 
 func (a *Aggregator) processTxs(txs []core.Tx) (commitments []core.Commitment, err error) {
 	var txsInCommitment []int
+
+	if len(txs) < int(a.cfg.TxsPerCommitment) {
+		return commitments, ErrNotEnoughTransactions
+	}
```

Comment:
Does a transaction take a long time to be submitted on chain if the commitment is unfilled?

Reply:
not really, it just might be non economical to submit unfilled commitments on-chain for a coordinator, so we allow him to configure that tightly.